packages/rOpenScPCA/R/calculate-clusters.R

```diff
@@ -0,0 +1,115 @@
+#' Calculate graph-based clusters from a provided matrix
+#'
```

Comment:
This is where I was suggestion you put the explanations of default & differences. Something like:

```suggestion
#' Calculate graph-based clusters from a provided matrix
#'
#' This function is provided to simplify application of some bluster functions for OpenScPCA data.
#' Note that defaults for some arguments may differ from the bluster defaults.
#' In particular, the clustering algorithm defaults to "louvain" and the distance measure to "jaccard" 
#' to align with common practice. 
```